test/kubemark/start-kubemark-master.sh

```diff
@@ -15,18 +15,25 @@
 # limitations under the License.
 
 # TODO: figure out how to get etcd tag from some real configuration and put it here.
+
+EVENT_STORE_IP=$(cat /tmp/event-store-ip)
+if [ "${EVENT_STORE_IP}" == "127.0.0.1" ]; then
```

Comment:
Do we really need to special case localhost like this? Can we not run it on port 4002 irrespective of whether it's local or not?